gym_homer/envs/homer_env.py

```diff
@@ -317,9 +320,15 @@ def save_results(self) -> None:
         results = pd.merge(obs, info, left_index=True, right_index=True)
         # Note for repetitions > 1 this will overwrite previous repetitions.
         device = self.device_id if self.device_id != None else "dummy"
-        results.to_csv(self.save_path+f"/{device}_results_array.csv", 
-                       index=False)
-    
+        if self.print_save:
+            print(results['updated_action'].value_counts())
+            results.to_csv(
+                self.save_path+f"/{device}_results_array_{datetime.now().strftime('%Y_%m_%d_%H_%M_%S')}.csv",
+                index=False
+            )
+        else:
+            results.to_csv(self.save_path+f"/{device}_results_array.csv", index=False)
+            
```

Comment:
This block saves the output of the the environment each time a done signal is issues, any time (train/test/val) the environment is done. 
The self.print_save is set at the __init__() block for the env and is not available as a flag in the config file.